git-imerge

```diff
@@ -3717,6 +3749,18 @@ def main(args):
             help='the tip of the branch to be merged into HEAD',
         )
 
+    def add_commit_to_reparent_argument(subparser):
+        subparser.add_argument(
+            '--commit-to-reparent', dest='commit_to_reparent', action='store', default=None,
```

Comment:
* I think it would be ok to name this option `--commit`.
* `dest` is redundant because that's the default.
* `action` is redundant because `'store'` is the default action.
* `default=None` is redundant because that is the default. However, don't you actually want `default='HEAD'` here? That would remove the need for `cmd_reparent` to check this argument and set it to `HEAD` if it was missing.